themes/gatsby-theme-blog-mdx/gatsby-config.js

```diff
@@ -0,0 +1,75 @@
+const path = require(`path`)
+
+module.exports = ({ defaultLayouts = {} } = {}) => {
+  const themeLayouts = {
+    posts: require.resolve(`./src/templates/post`),
+    default: require.resolve(`./src/components/layout`),
```

Comment:
since default applies to all MDX files ever, we probably shouldn't use it in themes.